e2e/pages/events/provideStatementOfServiceEvent.page.js

```diff
@@ -0,0 +1,70 @@
+const I = actor();
+const postcodeLookup = require('../../fragments/addressPostcodeLookup');
+
+module.exports = {
+  fields: function (index) {
+    return {
+      recipients: {
+        name: `#statementOfService_${index}_name`,
+        addressCheck: `#statementOfService_${index}_addressCheck-Yes`,
+        address: `#statementOfService_${index}_address_address`,
+        documents: `#statementOfService_${index}_documents`,
+        date: {
+          day: `#statementOfService_${index}_date-day`,
+          month: `#statementOfService_${index}_date-month`,
+          year: `#statementOfService_${index}_date-year`,
+        },
+        timeSent: `#statementOfService_${index}_timeSent`,
+        sentBy: {
+          group: `#statementOfService_${index}_sentBy`,
+          email: `#statementOfService_${index}_sentBy-EMAIL`,
+          post: `#statementOfService_${index}_sentBy-POST`,
+          givenInPerson: `#statementOfService_${index}_sentBy-GIVEN_IN_PERSON`,
+        },
+        emailAddress: `//*[@id="statementOfService_${index}_email"]`,
+        post: `#statementOfService_${index}_postOfficeAddress`,
+      },
+    };
+  },
+
+  addRecipientButton: '//*[@id="statementOfService"]/div/button[1]',
+
+  addRecipient() {
+    I.click(this.addRecipientButton);
+    I.wait(1);
+  },
+
+  async enterRecipientDetails(recipients) {
```

Comment:
Can we split this method up into smaller more focus methods e.g enterReceipentsAddress etc

Reply:
I think the form is small enough it is not worth to split up and is better to keep it as one flowing form.